lib/authentication/auth_web.js

```diff
@@ -9,6 +9,8 @@ var net = require('net');
 var querystring = require('querystring');
 var URLUtil = require('./../../lib/url_util');
 
+const WAIT_FOR_BROWSER_ACTION_TIMEOUT = 120000;
```

Comment:
it should be part of global config

Reply:
It isn't set by global config. This timeout is only in connection options and constant default is inside Authenticator class.